doc/relation_extraction.md

```diff
@@ -57,7 +57,7 @@ DeepDive makes good use of uncertainty to improve predictions during the
 [probabilistic inference](inference.md) step. For example, DeepDive may figure
 out that a certain mention of "Barack" is only 60% likely to actually refer to
 "Barack Obama", and use this fact to discount the impact of that mention on the
-final result for the entity "Barack Obama". DeepDive can also make use the
-domain knowledge, and allows users to encode rules such as "If Barack is married
+final result for the entity "Barack Obama". DeepDive can also make use of 
```

Comment:
Hey Jaeho,

Yeah, sorry. I'm still trying to figure out how to make edits properly. I
was doing them all inside git's UI; think I'll just move to the command
line in the future. I saw that two didn't pass Travis, and I figured it was
because of trailing white space, so I closed them. I'll go in and get rid
of the whitespace offenders and resubmit the pull request. Thanks!

On Tue, Feb 9, 2016 at 5:43 PM, Jaeho Shin <notifications@github.com> wrote:

> In doc/relation_extraction.md
> <https://github.com/HazyResearch/deepdive/pull/484#discussion_r52405088>:
>
> > @@ -57,7 +57,7 @@ DeepDive makes good use of uncertainty to improve predictions during the
> >  [probabilistic inference](inference.md) step. For example, DeepDive may figure
> >  out that a certain mention of "Barack" is only 60% likely to actually refer to
> >  "Barack Obama", and use this fact to discount the impact of that mention on the
> > -final result for the entity "Barack Obama". DeepDive can also make use the
> > -domain knowledge, and allows users to encode rules such as "If Barack is married
> > +final result for the entity "Barack Obama". DeepDive can also make use of
>
> probably the space at the end of this line
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HazyResearch/deepdive/pull/484/files#r52405088>.
>